pkg/gbpserver/crd_mo.go

```diff
@@ -0,0 +1,105 @@
+/***
+Copyright 2020 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// GBP Mo hooks
+
+package gbpserver
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+// CRDMo acts as the interface between a CRD and the corresponding MO.
+// Implement this interface in the CRD watcher
+type CRDMo interface {
```

Comment:
Can we rename to something like OpflexMo or GenieMo?

Reply:
That's too generic. All MOs are Opflex/Genie MOs. How about CustomMo or FeatureMo? @tomflynn

Reply:
I figured this interface *could* be used for any opflex MO but I guess something like CustomOpflexMo is fine.